packages/cyberstorm/src/components/TextInput/TextInput.tsx

```diff
@@ -4,14 +4,19 @@ import styles from "./TextInput.module.css";
 import { Icon } from "../Icon/Icon";
 import { classnames } from "../../utils/utils";
 
-export interface TextInputProps {
+export interface TextInputProps
+  extends React.ComponentPropsWithRef<"input">,
+    React.PropsWithChildren {
```

Comment:
To clarify my "what" comment from the earlier review: "what in tarnation?"

Seriously though, why accept children? Doesn't seem they're used anywhere in the context of this PR? Also IDK what the use case for passing children to an input would be? Pls remove or justify the need for this in a review comment (and possibly code comment as well).